### DISCUSSION TITLE
NuGet Packages missing

### DISCUSSION THREAD

**Discussion by ZadePlayzz** [winstxnhdw/lc-hax#491] [General]
What would cause Microsoft SDKs folder and NuGet packages to not be found when launching requirements? I managed to fix the issue by creating a new Microsoft SDks folder in the programs x86 and finding the .nuget folder and renaming it NuGetPackages and placing it inside.

**Comment by winstxnhdw** [comment]
Not sure either. I am guessing these people are following some guide that tells them to install Visual Studio and build, which screws up their system dependencies. You can see #123.

**Comment by winstxnhdw** [comment]
I made a change in the latest commit that should hopefully solve this issue. Let me know if it works.

**Reply by ZadePlayzz** [reply]
Okay thank you that kinda explains it. i deleted something to do with visual studio and it screwed it up somehow. im new to this. thank you for your time its much appreciated.
